test/jdk/java/awt/print/PrinterJob/SecurityDialogTest.java

```diff
@@ -21,210 +21,108 @@
  * questions.
  */
 
-/**
+import java.awt.Frame;
+import java.awt.print.PageFormat;
+import java.awt.print.PrinterJob;
+import java.lang.reflect.InvocationTargetException;
+
+import javax.print.PrintService;
+import javax.print.attribute.HashPrintRequestAttributeSet;
+import javax.print.attribute.PrintRequestAttributeSet;
+import javax.swing.BorderFactory;
+import javax.swing.Box;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.SwingUtilities;
+
+/*
  * @test
  * @bug 4937672 5100706 6252456
  * @key printer
- * @run main/othervm/manual -Djava.security.manager=allow SecurityDialogTest
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual/othervm -Djava.security.manager=allow SecurityDialogTest
  */
+public class SecurityDialogTest extends Frame {
+    private static final String INSTRUCTIONS =
```

Comment:
`INSTRUCTIONS` can be moved inside main.

Reply:
Most of the test with PassFailJFrame instruction stored as string inside class, what additional benefit will get keeping it inside Main ?

Reply:
When it can become local variable then why to keep it in class? Not like significant benefit maybe, but yeah just to keep its scope locally when it can be.

Reply:
> INSTRUCTIONS can be moved inside main.

No. Well, it can but what's the point?

I find a value in having the instructions as a dedicated constant at the top of the file. When you open the test, the instructions will be the first thing you see, and it's what you need after reading the summary in `@summary` tag above the class declaration.